packages/Discretization/test/tstMeshGenerator.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include "MeshGenerator.hpp"
 #include <Teuchos_UnitTestHarness.hpp>
+#include <fstream>
```

Comment:
@Rombur Got a compile error because that header was missing.  Piggybacking on that PR.